flow-typed/types/log-level.js

```diff
@@ -2,3 +2,9 @@
 declare type PKLogLevelObject = { value: number, name: string };
 declare type PKLogLevels = { [level: string]: PKLogLevelObject };
 declare type PKLogLevelTypes = { [level: string]: string };
+declare type LogHandlerType = (messages: any[], context: Object) => void;
+declare type PKLogConfigObject = {
+  level:string,
+  playerVersion:boolean,
+  handler:LogHandlerType
```

Comment:
I'm not flow but seems these should be `?boolean` and `?LogHandlerType`   :)

Reply:
fixed